Paper: Algorithms to Determine Asteroid’s Physical Properties using Sparse and Dense Photometry, Robotic Telescopes and Open Data

Spacegirl123 opened the pull request:
If you are creating this PR in order to submit a draft of your paper, please name your PR with `Paper: <title>`. An editor will then add a `paper` label and GitHub Actions will be run to check and build your paper.

See the [project readme](https://github.com/scipy-conference/scipy_proceedings#preview-your-paper) for more information.

_Editor_: <!--editor-->Chris Calloway @cbcunc<!--end-editor-->

_Reviewers_: <!--reviewers-list-->
- Chuchu Wu @Chuchugo
- Kalyan Prasad @kalyan678
<!--end-reviewers-list-->

Spacegirl123 committed:
SciPy 2024 paper

kalyan678 commented:
@Spacegirl123 - Hello, My name is Kalyan , a Data and AI scientist from India. With extensive experience in tackling machine learning and artificial intelligence challenges at the enterprise level across diverse domains, I am thrilled to review your paper for the SciPy Conference. This is my second year in a row reviewing for the conference, and I look forward to offering valuable suggestions and inputs to refine your paper, ensuring it effectively reaches and benefits a broader audience.

kalyan678 commented:
Hi @Spacegirl123 Below are some points that you can incorporate in order to improve the abstract. Let me know if you have any questions.
1. I think it is good to explain briefly why understanding near-Earth asteroids is important. 
2. What's the advantage of being open source algorithm. Write a short line about it.
3. I think it is also good to describe how your algorithms combine continuous and infrequent light data.
4. Try simplifying or briefly explaining terms like "absolute magnitude," "rubble-pile strength," and "S-type composition" for broader understanding.
5. It is good to mention how the reduced orbital period measurement contributes to planetary defense efforts.

# Introduction & Goals:
The introduction stays on track with its story. It explains the difficulties and aims clearly, without making things more complicated than they need to be. Good job!

If possible simplify terms like "phase curves" and provide quick explanations.

Regarding the goals, they are well-defined and commendable. However, it would be beneficial to elaborate more on how the developed methodology (PhAst) will be used in planetary defense tests such as NASA’s DART mission.

Additionally, please clarify how characterizing asteroid populations will help us better understand the diversity of compositions in the solar system.



# Methodology :
The methodology section is clear and concise, however I think you can describe how the algorithm integrates different data types. Include a brief overview or diagram of the workflow to make the process more transparent. 

# Data Integration:
In my opinion, It is good to explain how data from different sources are combined.

# Observational Process:
Overall the process details are good. Consider adding diagrams or flowcharts to visually represent the observational process, which can help readers better understand the sequence and interaction of steps.


# Case Study- Didymos Binary Asteroid: 
The initial observations of the Didymos binary asteroid are well-detailed, including its size, albedo, rotation period, and composition. This gives a good starting point for understanding the asteroid's properties. Well done!

# Results:
Please clarify if the processing time includes data retrieval and pre-processing? Also, provide a brief context for the listed physical properties for the NASA LUCY and UAE missions' targets to show their relevance.

# Future work:
The future applications of PhAst are promising. May be we can mention specific benefits or new opportunities these applications might bring.

# General Suggestions:

* Avoid or simplify technical terms where possible. Provide brief explanations for any unavoidable technical jargon.

* Consider adding diagrams, flowcharts, or graphs to make complex processes and data patterns clearer.

 * Try to use specific examples or scenarios to illustrate key points, such as how asteroid strength is determined or the implications of the taxonomy data.

* Please ensure a consistent narrative flow and structure throughout the sections for easier understanding and readability.


This concludes my initial review. Happy to clarify/ discuss if you have any questions on my comments. Good luck!

kalyan678 commented:
Hi @Spacegirl123 - The introduction stays on track with its story. It explains the difficulties and aims clearly, without making things more complicated than they need to be. Good job!

If possible simplify terms like "phase curves" and provide quick explanations.

Regarding the goals, they are well-defined and commendable. However, it would be beneficial to elaborate more on how the developed methodology (PhAst) will be used in planetary defense tests such as NASA’s DART mission.

Additionally, please clarify how characterizing asteroid populations will help us better understand the diversity of compositions in the solar system.

cbcunc commented:
Review reminders sent to @chuchugo and @kalyan678

chuchugo commented:
Hi @Spacegirl123 Thanks for your paper. After reading the paper, here are some suggestions regarding the improvement. Hope you find it helpful.

# General Comments

- The paper discusses measurements on phase angles of asteroids. The author combines sparse photometry from surveys and dense photometry from citizens to determine asteroids' albedo, size, rotation, strength, and composition. Didymos binary asteroid and combined them with GAIA etc. surveys. Measured 35-minute decrease in the mutual orbital period after impact by the DART mission.

- The paper seems to be incomplete. Please explain your algorithm, figures, mathematical definitions, equations, etc.. If a key part of the algorithm has already been published, please summarize the key steps and give a full citation in the references. The goal is to make the paper reproducible. Without completeness and reproducibility, it is impossible to review it and also difficult to learn from it. Hope it can be revised in a more solidated way since you probably have a nice algorithm that astronomers can use. I will give some specific comments below, but they all point to this general comment.


# Specific Comments

- Organization:
  - The organization is a little difficult to follow as a research paper, and is more written like a blog post. For instance, the definition "Dense and Sparse Photometry" does not need to be a separate section. Same with "Goals" - these should both be in the introduction.
  - Banner figure axes were not labeled
  - The references are confusing. Please use a standardized reporting format, such as IEEE, and reference them one by one using brackets [1], [2] etc.

- Verifiability:
  - Many claims are unsubstantiated. For instance, "Asteroid strength can be inferred from the rotation period. If the rotation period is less than 2.2 hours, the asteroid must be a strength-bound single rock. Otherwise, it would fly apart."  How was this determined? What other sources use this criteria? What fundamental equations point to this requirement?

- Computational content and Technical content: 
  - "Over 58% of the asteroids characterized by PhAst are carbonaceous, showing they are the most abundant type in our Solar System." -- Is this a new discovery you are presenting, or is this a generally known fact? It seems rather odd to present such a fundamental finding in an off-hand way in the middle of a conference presentation.
  - Please include the relevant data, key algorithm steps, and the code so it can be validated

- Clarity:
  - Most readers will not be subject-matter experts on astronomy. As such, please define key astronomical terms such as "albedo", H, G.
  - The introduction does not adequately survey the field showing other alternatives and recent progress toward solving the stated problem problem (integrating sparse and dense datasets)
  - Please explain the figures. I cannot find references to any of the figures in the paper. Nor can I find captions for the figures

cbcunc commented:
@chuchugo and @kalyan678 you may also open a review on this PR to line by line make suggestions and comments more specifically. That way when the author resolves a review comment or suggestion, we can know how much progress is being made to address your concerns.

kalyan678 commented:
> @chuchugo and @kalyan678 you may also open a review on this PR to line by line make suggestions and comments more specifically. That way when the author resolves a review comment or suggestion, we can know how much progress is being made to address your concerns.

@cbcunc - I have already provided my initial  review section wise as per the paper. I hope that helps the author to understand the context and work accordingly. Let me know if there are any questions!

Spacegirl123 committed:
Update main.md

Spacegirl123 commented:
Hello,

I have addressed all the reviews in my updated paper. I wanted to change my image files and banner since they do not match with my text, but GitHub is not letting me remove them/add new ones. The edit and delete buttons are greyed out. Could you let me know how I can change them?

Best regards,
Arushi

fwkoch commented on papers/Arushi_Nath/main.md:
```diff
@@ -0,0 +1,222 @@
+title: Algorithms to Determine Asteroid’s Physical Properties using Sparse and Dense Photometry, Robotic Telescopes and Open Data
+abstract: |
+  The rapid pace of discovering asteroids due to advancements in detection techniques outpaces current abilities to analyze them comprehensively. Understanding an asteroid's physical properties is crucial for effective deflection strategies and improves our understanding of the solar system's formation and evolution. Dense photometry provides continuous time-series measurements valuable for deter ...
+Applying the algorithms to the Didymos binary asteroid, combined with data from GAIA, the Zwicky Transient Facility, and ATLAS photometric sky surveys, revealed Didymos to be 840 meters wide, with a 0.14 albedo, an 18.14 absolute magnitude, a 2.26-hour rotation period, rubble-pile strength, and an S-type composition. Didymos was the target of the 2022 NASA Double Asteroid Redirection Test (DART)  ...
+This work empowers citizen scientists to become planetary defenders, contributing significantly to planetary defense and enhancing our understanding of solar system composition and evolution.
```
Fixing up this indentation should allow the checks to pass!

```suggestion
  Applying the algorithms to the Didymos binary asteroid, combined with data from GAIA, the Zwicky Transient Facility, and ATLAS photometric sky surveys, revealed Didymos to be 840 meters wide, with a 0.14 albedo, an 18.14 absolute magnitude, a 2.26-hour rotation period, rubble-pile strength, and an S-type composition. Didymos was the target of the 2022 NASA Double Asteroid Redirection Test (DART) mission. The algorithm successfully measured a 35-minute decrease in the mutual orbital period following the DART mission, equating to a 40-meter reduction in the mutual orbital radius, proving a successful deflection. Analysis of the broader asteroid population highlighted significant compositional diversity, with a predominance of carbonaceous (C-type) asteroids in the outer regions of the asteroid belt and siliceous (S-type) and metallic (M-type) asteroids more common in the inner regions. These findings provide insights into the diversity and distribution of asteroid compositions, reflecting the conditions and processes of the early solar system. 
  This work empowers citizen scientists to become planetary defenders, contributing significantly to planetary defense and enhancing our understanding of solar system composition and evolution.
```

Spacegirl123 committed:
Update papers/Arushi_Nath/main.md

Co-authored-by: Franklin Koch <franklin@curvenote.com>

cbcunc commented:
> Hello,
> 
> I have addressed all the reviews in my updated paper. I wanted to change my image files and banner since they do not match with my text, but GitHub is not letting me remove them/add new ones. The edit and delete buttons are greyed out. Could you let me know how I can change them?
> 
> Best regards, Arushi

@Spacegirl123 Do you mean you are not able to make new commits to your [Spacegirl123:2024](/Spacegirl123/scipy_proceedings/tree/2024) branch?

cbcunc commented:
@Spacegirl123 ^^^

Spacegirl123 commented:
> > Hello,
> > I have addressed all the reviews in my updated paper. I wanted to change my image files and banner since they do not match with my text, but GitHub is not letting me remove them/add new ones. The edit and delete buttons are greyed out. Could you let me know how I can change them?
> > Best regards, Arushi
> 
> @Spacegirl123 Do you mean you are not able to make new commits to your [Spacegirl123:2024](/Spacegirl123/scipy_proceedings/tree/2024) branch?

I am able to make commits, but not to the .png files. See attached.
![ScipyGit](https://github.com/user-attachments/assets/8f3447df-db79-4526-9eba-f5e3bcfd48d6)

fwkoch commented:
@Spacegirl123 - If you want to delete the file using the web interface, you must go the branch in your fork and delete from over there:

![image](https://github.com/user-attachments/assets/d901a42b-07ae-446b-81bb-638de96783c8)

Spacegirl123 committed:
Delete papers/Arushi_Nath/figure1.png

Spacegirl123 committed:
Delete papers/Arushi_Nath/figure2.png

Spacegirl123 committed:
Add files via upload

Spacegirl123 commented:
> @Spacegirl123 - If you want to delete the file using the web interface, you must go the branch in your fork and delete from over there:
> 
> ![image](https://private-user-images.githubusercontent.com/9453731/350829279-d901a42b-07ae-446b-81bb-638de96783c8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE2OTE3NTIsIm5iZiI6MTcyMTY5MTQ1MiwicGF0aCI6Ii85NDUzNzMxLzM1MDgyOTI3OS1kOTAxYTQyYi0wN2FlLTQ0NmItODFiYi02MzhkZTk2NzgzYzgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcyMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MjJUMjMzNzMyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9M2VmMTA2Y2EzZjNhMTg0YTEzYWUyZDBjZmM3ZjA1NzVlODZhYzBiNDM4YWQxY2FlN2ZjMDgzZmM5ZGI0YjhkYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.L6bUlbvEYMZlIVuFRyYU6sTdgpgQQZmrmhKB_o-dCpU)

Thanks. I have edited all my figures.

kalyan678 reviewed (approved):
Apologies for responding late. I can see that you’ve made the changes based on the feedback, and they look great. Overall, it's excellent work. Wishing you the best of luck moving forward!
